native/src/main/scala/sqs4s/api/lo/CreateQueue.scala

```diff
@@ -0,0 +1,85 @@
+package sqs4s.api.lo
+
+import cats.effect.{Clock, Sync}
+import cats.implicits._
+import org.http4s.client.Client
+import org.http4s.scalaxml._
+import org.http4s.{Method, Request, Uri}
+import sqs4s.api.SqsSetting
+import sqs4s.api.errors.SqsError
+import sqs4s.internal.aws4.common._
+import sqs4s.internal.models.CReq
+import CreateQueue.defaults._
+
+import scala.concurrent.duration._
+import scala.xml.{Elem, XML}
+
+case class CreateQueue[F[_]: Sync: Clock](
+  name: String,
+  delay: Duration = DelaySeconds,
+  maxMessageSize: Int = MaxMessageSize,
+  messageRetentionPeriod: Duration = MessageRetentionPeriod)
+    extends Action[F, String] {
+
+  def runWith(setting: SqsSetting)(implicit client: Client[F]): F[String] = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)